src/index.html

```diff
@@ -4,9 +4,304 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="index.html" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__item">
+            <a href="index.html" class="header__link">Samsung</a>
+          </li>
+          <li class="header__item">
+            <a href="index.html" class="header__link">Smartphones</a>
+          </li>
+          <li class="header__item">
+            <a
+              href="index.html"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="index.html" class="header__link">Gadgets</a>
+          </li>
+          <li class="header__item">
+            <a href="index.html" class="header__link">Tablets</a>
+          </li>
+          <li class="header__item">
+            <a href="index.html" class="header__link">Photo</a>
+          </li>
+          <li class="header__item">
+            <a href="index.html" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+
+      <section class="cards">
```

Comment:
```suggestion
    <main>
      <section class="cards">
```
Don't leave empty lines between parent and child elements